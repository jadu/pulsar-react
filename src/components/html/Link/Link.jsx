import React from 'react';

export default class Link extends React.Component {

  render() {
    let {
      children,
      ...props
    } = this.props;

    return <a {...props}>{children}</a>;
  }
}
