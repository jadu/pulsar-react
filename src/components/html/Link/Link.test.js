'use strict';

import React from 'react';
import { render } from 'enzyme';

import Link from './Link';

it('renders the base Link', () => {
  const tree = render(
    <Link href="#foo">bar</Link>
  );
  expect(tree).toMatchSnapshot();
});
