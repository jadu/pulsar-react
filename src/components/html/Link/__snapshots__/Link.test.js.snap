// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the base Link 1`] = `
<a
  href="#foo"
>
  bar
</a>
`;
